draft.md

```diff
@@ -0,0 +1,12 @@
+# An value-type for `C++`
+
+ISO/IEC JTC1 SC22 WG21 Programming Language `C++`
+
+P1950R0
+
+Working Group: Library Evolution, Library
+
+_Jonathan Coe \<jonathanbcoe@gmail.com\>_
+
+_Anthony Peacock \<sparent@adobe.com\>_
```

Comment:
Its "Antony" (no 'h' in my name) and the email address is ant.peacock@gmail.com. Not sure Sean Parent wants to get emails addressed to me ;)